src/test/java/emissary/util/ShortNameComparatorTest.java

```diff
@@ -37,6 +37,15 @@ public void testNonsenseNames() {
         assertEquals("Bad components sort in alpha order", "foo-att-abc", l.get(0).shortName());
     }
```

Comment:
This comment is separate from the changes in this PR, but `testOrdering` and `testImplComparator` appear to be identical tests.